<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgfreealign</class>
 <widget class="QDialog" name="dlgfreealign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="chXZ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Grow X,CX dimension to Window Width</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chXL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock X to Right Window Side</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Free Buttons GROW on Window SIZE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chYZ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Grow Y,CY dimension to Window Width</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chYL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock Y to Down Window Side</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chX2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock X2 to Right Window Side</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chY2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock Y2 to Down Window Side</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chXM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move X analog to Window Width</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chYM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Y analog to Window Width</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
